cargo-shuttle/src/main.rs

```diff
@@ -1,11 +1,18 @@
 use anyhow::Result;
 use cargo_shuttle::{Args, CommandOutcome, Shuttle};
 use clap::Parser;
+use std::{env, path::PathBuf};
 
 #[tokio::main]
 async fn main() -> Result<()> {
     env_logger::init();
 
+    env::var("SHELL").ok().map(|x| {
+        PathBuf::from(x)
+            .file_name()
+            .map(|x| env::set_var("SHELL", x))
+    });
```

Comment:
Why is this needed?

Reply:
Since the `FromStr` commit code has been removed, I think it is safe to remove this too right?

Reply:
For the panic, I think that line was correct as `env!("CARGO_PKG_NAME")` since it should get the name of cargo-shuttle and not the project if I understand correctly :man_shrugging:

Reply:
The only thing still blocking this PR for is modifying the `SHELL` env var since it will be an unexpected side effect (seeing that shuttle is not a shell utility). I think this should rather be fixed as a PR to `clap_complete` to not introduce these lines to shuttle